dist/tools/kconfiglib/README.md

```diff
@@ -0,0 +1,21 @@
+# kconfiglib
+
+## Examples
+
+The examples folder contains examples of design patterns that can be used along
+with some information of how and when to use them.
+
+## Symbol Conventions
+
+- `ERROR_*` - If this symbol is selected, it will throw an error in the build system to indicate something is configured wrong. Used when the base kconfig tool is not expressive enough.
```

Comment:
![giphy](https://user-images.githubusercontent.com/5381296/213707129-42784e0a-7d53-4947-9823-33f5e8e4202a.gif)

https://doc.riot-os.org/kconfig-in-riot.html#autotoc_md2020 do we need it again?

Reply:
Just needed to write it somewhere, I will add it to the docs